EventSubscriber/FrontendControllerSubscriber.php

```diff
@@ -82,6 +83,12 @@ public function onKernelRequest(GetResponseEvent $event): void
                 $frontendController->settingLanguage();
                 $frontendController->settingLocale();
             }
+
+            if (!$frontendController->tmpl->setup && ($site = $event->getRequest()->attributes->get('site')) instanceof SiteInterface) {
```

Comment:
Check with count if array is not empty. Assigned outside of if.

Reply:
Okay